.travis.yml

```diff
@@ -8,19 +8,33 @@ cache:
 matrix:
   include:
     - node_js: "8"
-      env: PERCY_ENABLE=1
-    - node_js: "7"
-      env: PERCY_ENABLE=0
-    - node_js: "6"
-      env: PERCY_ENABLE=0
+      env: LINT=true
     - node_js: "4"
-      env: PERCY_ENABLE=0
+      env: UNIT_TEST=true COVERAGE=false
+    - node_js: "6"
+      env: UNIT_TEST=true COVERAGE=false
+    - node_js: "7"
+      env: UNIT_TEST=true COVERAGE=false
+    - node_js: "8"
+      env: UNIT_TEST=true COVERAGE=true
+    - node_js: "8"
+      env: INTEGRATION_TEST=react-percy-storybook PERCY_PROJECT=percy/react-percy-storybook PERCY_TOKEN="$STORYBOOK_PERCY_TOKEN"
```

Comment:
Couldn't find a better way to set the percy token on matrix jobs, so had to just add global variables in Travis and reference them this way. @fotinakis let me know if you had a better scheme in mind!